### DISCUSSION TITLE
Link in SPFX Property Pane header

### DISCUSSION THREAD

**Discussion by claytonEtillman** [SharePoint/sp-dev-docs#7458] [Q&A] [answered]
Hi, how do you render a link or other HTML tags in an SPFX Property Pane header as shown in this image: docs/images/property-pane-example.png

![image](https://user-images.githubusercontent.com/19317417/139350420-1438a36d-1dd8-4d36-862e-cda55b3add28.png)

Thanks!

**Comment by AJIXuMuK** [comment, marked as answer]
I would use [Markdown content](https://pnp.github.io/sp-dev-fx-property-controls/controls/PropertyPaneMarkdownContent.html) property field.

**Reply by claytonEtillman** [reply]
Awesome! Thank you very much, Alex.
